analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -100,6 +103,9 @@
     <Content Include="CBDEMocks\CBDEWaitAndSucceeds.exe">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="ResourceTests\SonarProjectConfig\**\*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```

Comment:
Can we use something like `ResourceTests\**\*` to be done with the topic for once and all and to remove the next 50 repetitive lines?

The pattern and purpose is pretty similar to test cases above
```
    <None Include="TestCases\**\*">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
```

Reply:
Awesome!